build.go

```diff
@@ -29,11 +29,11 @@ import (
 
 var (
 	// buildContainer is a docker container used to build go binaries
-	buildContainer = "golang:1.14.3"
+	buildContainer = "golang:1.12.17"
```

Comment:
I think it might be legitimate to go through and bump golang on master, it's just we should be careful about the backports.

Reply:
Oh, ok. I'll leave the changes in the master branch only then.